techmatters/flex-plugins
# Update scripts to generate flex-plugins deployment file

Primary reviewer: @stephenhand 

## Description
This PR:
- Updates the script that generates flex-plugins deployment file
- Remove code related to `createTwilioResources`, since we're doing this with Terraform now

### Related Issues
Fixes https://bugs.benetech.org/browse/CHI-1045

### Verification steps
- Fill .env with a testing HELPLINE, SHORT_HELPLINE and ENVIRONMENT
- From `scripts/` folder, run: `npm run generateDeploymentFiles`
- Check that the generated file match the desired code
